src/data/ab-attrs/stat-multiplier-ab-attr.ts

```diff
@@ -54,9 +55,9 @@ export class StatMultiplierAbAttr extends StatStageAbAttr {
     stat: BattleStat,
     statValue: NumberHolder,
     move: Move,
-    _target?: Pokemon,
+    target?: Pokemon,
   ): boolean {
-    if (stat === this.stat && (!this.condition || this.condition(pokemon, null, move))) {
+    if (stat === this.stat && (!this.condition || this.condition(pokemon, target!, move))) {
```

Comment:
I'm not a fan of the `!`.
Isn't a fallback to `null` an option? 
```suggestion
    if (stat === this.stat && (!this.condition || this.condition(pokemon, target ?? null, move))) {
```